Kudu.Contracts/Settings/DeploymentSettingsExtension.cs

```diff
@@ -249,5 +249,10 @@ public static bool DoBuildDuringDeployment(this IDeploymentSettingsManager setti
             // returning true by default here as an indicator of generally expected behavior
             return value == null || StringUtils.IsTrueLike(value);
         }
+
+        public static bool RunFromLocalZip(this IDeploymentSettingsManager settings)
+        {
+            return settings.GetValue(SettingsKeys.RunFromZip) == "1";
```

Comment:
We should probably make the deployment fail if this is set to a URL?

Reply:
Agreed, that would not make sense.